PoseEstimation-TFLiteSwift/ViewController.swift

```diff
@@ -9,12 +9,11 @@
 import UIKit
 
 class ViewController: UIViewController {
+    
+    let poseEstimator: PoseEstimator = PoseNetPoseEstimator()
```

Comment:
What is the mean of NetPose?

Reply:
[`PoseNet`](https://arxiv.org/abs/1505.07427) is the paper for pose estimation distributed in 2015. `PoseNetPoseEstimator` stand for `PoseNet` + `PoseEstimator` which means that the pose estimator implementing `PoseNet` tflite model.

Reply:
Thank, I undetstood.